File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntPluginMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdPlugin;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntPluginMap extends StudioBase
{
  public MetaIdPlugin[] keys;

  public Map<MetaIdPlugin, StudioEntPlugin> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntPluginMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import java.util.Map
import com.neome.api.meta.base.Types.MetaIdPlugin
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioEntPlugin

interface StudioEntPluginMap : StudioBase
{
  val keys: Array<MetaIdPlugin>
  val map: Map<MetaIdPlugin, StudioEntPlugin>
}